benchmark/scripts/run_smoke_bench

```diff
@@ -119,6 +119,8 @@ def test_opt_levels(args):
                             args.platform, output_file):
             changes = True
 
+    check_added(args, output_file)
```

Comment:
This is now called by default - which is fine.
So the -check-added option is obsolete, right?
Can you replace -check-added with a -no-check-added option which disables the checking? This would be useful for running the script locally to check compiler changes (and not benchmark changes).

Reply:
Makes sense. I'm on it...